### PR TITLE
added N/A for negative values as well as missing data

### LINKED ISSUE #1684: Correctly handle number of sales
When aquarius returns `-1` it means it couldn't fetch the number of orders.
In this case we should display `n/a`

image

### PR DESCRIPTION
**Pull request by EnzoVezzaro**
- Fixes #1684 .

Changes proposed in this PR:

- added N/A for negative values as well as missing data

### CODE REVIEW

**Comment by mihaisc** [comment]
add the fix on the teaser as well and check if we display it in other places , like profile page

image

**Comment by bogdanfazakas** [comment]
> add the fix on the teaser as well and check if we display it in other places , like profile page

Yes we have the issue in the profile too
Screenshot 2022-09-14 at 15 30 59

**Comment by EnzoVezzaro** [comment]
made changes when there's sales numbers. Hopefully I covered all components with these numbers.

File: src/components/@shared/Price/Conversion.tsx [hunk 1]
```diff
@@ -27,19 +27,21 @@ export default function Conversion({
   // referring to Coingecko tokenId in Prices context provider
   const priceTokenId = getCoingeckoTokenId(symbol)
 
+  console.log(price)
```
Comment (bogdanfazakas) [inline src/components/@shared/Price/Conversion.tsx:30]: missed this 😄

**Review by bogdanfazakas — APPROVED** [review]
LGTM, i remembered that we have decided to have prices as strings and send them between various services like that to avoid further issues, but as far i can see this update to Number affects just UI elements so we should be fine

File: src/components/@shared/Price/Conversion.tsx [hunk 2]
```diff
@@ -31,15 +31,15 @@ export default function Conversion({
     if (
       !prices ||
       !price ||
-      price === '0' ||
+      price === 0 ||
```
Comment (catalin8v) [inline src/components/@shared/Price/Conversion.tsx:34]: Screenshot 2022-09-17 at 10 49 47

this line is not necessary anymore :)

Comment (EnzoVezzaro) [reply src/components/@shared/Price/Conversion.tsx]: good catch, thanks 😊

**Pull request merged**